coursestyle.css

```diff
@@ -0,0 +1,7 @@
+.body {
+	font-family: 'Montserrat', sans-serif;
+}
+
+.header__title {
+	font-family: 'Oswald', sans-serif;
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/shiny-tribble/pullRequest?prid=5485358)